### PR TITLE
Cucumber Runner

### PR DESCRIPTION
**Pull request by Claudia-20**

### CODE REVIEW

File: src/test/resources/feature/ManageItemsToShoppingCart.feature [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    And the following fields are updated
+      | Total products | $87.02 |
+      | Total          | $89.02 |
+      | TOTAL          | $89.02 |
+    When user press on '-' button for item 'Faded Short Sleeve T-shirts'
+    Then the quantity of item is changed to 1
+    And the following fields are updated
+      | Total products | $70.51 |
+      | Total          | $72.51 |
+      | TOTAL          | $72.51 |
```
Comment (venache1) [inline src/test/resources/feature/ManageItemsToShoppingCart.feature]: Please remove feature file, and add it to feature file task!

File: src/test/java/com/endava/soa4/RunCucumberTest.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+import io.cucumber.junit.Cucumber;
+import io.cucumber.junit.CucumberOptions;
+import org.junit.runner.RunWith;
+
+@RunWith(Cucumber.class)
+@CucumberOptions(
+        features = "resources/feature",
+        glue = {"java/com/endava/soa4/stepDefinition"},
+        tags = "@Claudia and @Irina and @Stefan and @Artiom and @Anatolie")
```
Comment (venache1) [inline src/test/java/com/endava/soa4/RunCucumberTest.java]: This means that this Runner will run only tests with all this tags present. Please change this, as you need to be present only one name and @Run tag

**Review by venache1 — APPROVED** [review]

File: src/test/resources/feature/ManageItemsToShoppingCart.feature (hunk 1 above)
Comment (Claudia-20) [reply src/test/resources/feature/ManageItemsToShoppingCart.feature]: Done

File: src/test/java/com/endava/soa4/RunCucumberTest.java (hunk 2 above)
Comment (Claudia-20) [reply src/test/java/com/endava/soa4/RunCucumberTest.java]: Done

**Review by venache1 — APPROVED** [review]

**Pull request merged**